openpype/hosts/maya/plugins/publish/extract_maya_scene_raw.py

```diff
@@ -97,15 +92,15 @@ def process(self, instance):
         self.log.info("Extracted instance '%s' to: %s" % (instance.name, path))
 
     @staticmethod
-    def _add_loaded_containers(members):
+    def _get_loaded_containers(members):
         # type: (list) -> list
         refs_to_include = [
             cmds.referenceQuery(ref, referenceNode=True)
             for ref in members
             if cmds.referenceQuery(ref, isNodeReferenced=True)
         ]
 
-        refs_to_include = set(refs_to_include)
+        members_with_refs = set(refs_to_include).union(members)
```

Comment:
Should we directly turn this into a set like this?
```python
        refs_to_include = {
            cmds.referenceQuery(ref, referenceNode=True)
            for ref in members
            if cmds.referenceQuery(ref, isNodeReferenced=True)
        }

        members_with_refs = refs_to_include.union(members)
```

Also I feel the variable name in the loop is confusing - it's not referring to a `reference` but to `node` that might be referenced or not.
```python
        refs_to_include = {
            cmds.referenceQuery(node, referenceNode=True)
            for nodein members
            if cmds.referenceQuery(node, isNodeReferenced=True)
        }

        members_with_refs = refs_to_include.union(members)
```
This makes more sense to me.